.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -42,17 +40,14 @@ If all is good, then in 24h, I will start the release. Thanks!
   - [ ] Create a new document for the release in the [releases folder](https://github.com/medic/cht-docs/tree/main/content/en/core/releases).
   - [ ] Ensure all issues are in the GH Milestone, that they're correctly labelled (in particular: they have the right Type, "UI/UX" if they change the UI, and "Breaking change" if appropriate), and have human readable descriptions.
   - [ ] Use [this script](https://github.com/medic/cht-core/blob/master/scripts/release-notes) to export the issues into our release note format.
```

Comment:
Here is just a recommendation to make everything easier. Maybe we can create a README.md file [here](https://github.com/medic/cht-core/blob/master/scripts/release-notes), with the instructions that are inside the script, in that way when someone opens the link the instructions will be shown immediately and there would not be room for doubts. Or maybe just add a comment in the step saying that the steps to run the script are inside the file. Or just edit the link so it will point directly to the file.

I know that by default or by instinct everyone will open the file and figure out how to run it. But when I was doing the process I was a little lost and in "alert mode" to try to make everything the best way possible, and I would have appreciated that everything was just in front of my eyes and I didn't have to assume or guess anything.

Reply:
Link updated.